packages/material-ui/src/Popper/Popper.test.js

```diff
@@ -71,19 +68,25 @@ describe('<Popper />', () => {
         out: 'top',
       },
     ].forEach((test) => {
-      it(`should ${test.in === test.out ? 'not' : ''}flip ${
+      it(`should ${test.in === test.out ? 'not' : ''} flip ${
         test.in
       } when direction=rtl is used`, () => {
-        render(
-          <ThemeProvider theme={rtlTheme}>
-            <Popper {...defaultProps} placement={test.in}>
-              {({ placement }) => {
-                return <div data-testid="placement">{placement}</div>;
-              }}
-            </Popper>
-            ,
-          </ThemeProvider>,
-        );
+        function Test() {
+          const [anchorEl, setAnchorEl] = React.useState(null);
+
+          return (
+            <ThemeProvider theme={rtlTheme}>
+              <div style={{ margin: '5em' }} ref={setAnchorEl} />
```

Comment:
Verified in Chrome 91 that we do need an `anchorEl` with dimensions: https://codesandbox.io/s/naughty-taussig-nfbr5?file=/src/Demo.tsx.

The assertions would fail with 
```suggestion
              <div style={{ display: 'inline' }} ref={setAnchorEl} />
```
which is basically what an unconnected `document.createElement('svg')`  would look like to popper.js